clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10748,7 +10748,7 @@ static void addArgs(ArgStringList &DstArgs, const llvm::opt::ArgList &Alloc,
   }
 }
 
-static bool supportDynamicLinking(const llvm::opt::ArgList &TCArgs) {
+static bool allowDeviceDependencies(const llvm::opt::ArgList &TCArgs) {
```

Comment:
Sorry. Can you please update this to allowDeviceImageDependencies to keep things consistent?

Thanks